webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py

```diff
@@ -43,7 +44,25 @@ async def test_subscribe(bidi_session, subscribe_events, inline, new_tab, wait_f
     await bidi_session.browsing_context.navigate(context=new_tab["context"], url=url)
     event = await on_entry
 
-    assert_navigation_info(event, new_tab["context"], url)
+    assert_navigation_info(event, {"context": new_tab["context"], "url": url})
+
+
+async def test_timestamp(bidi_session, current_time, subscribe_events, inline, new_tab, wait_for_event):
+    await subscribe_events(events=[DOM_CONTENT_LOADED_EVENT])
+
+    time_start = await current_time()
+
+    on_entry = wait_for_event(DOM_CONTENT_LOADED_EVENT)
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(context=new_tab["context"], url=url)
+    event = await on_entry
+
+    time_end = await current_time()
+
+    assert_navigation_info(
+        event,
+        {"context": new_tab["context"], "timestamp": int_interval(time_start, time_end)}
```

Comment:
there is no assertion of the `timestamp` in the `assert_navigation_info`.

Reply:
Oh, good point! I'll get that missing assertion added. Looks like it went through our review unnoticed. Thanks.